### PR TITLE
Fix for issue #1487

### PR DESCRIPTION
**Pull request by vijayviji**
Created a separated VM environments for developers and users. Also, fixed a minor issue in bootstrap.sh.

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -55,9 +55,9 @@ dokku_install_package() {
   apt-get update -qq > /dev/null
 
   if [[ -n $DOKKU_CHECKOUT ]]; then
-    apt-get install dokku=$DOKKU_CHECKOUT
+    apt-get install -y dokku-$DOKKU_CHECKOUT
   else
-    apt-get install dokku
+    apt-get install -y dokku
```
Comment (josegonzalez) [inline bootstrap.sh]: Why the changes here?

File: Vagrantfile [hunk 2]
```diff
@@ -30,15 +29,27 @@ Vagrant::configure("2") do |config|
 
   config.vm.define "empty", autostart: false
 
+  # For normal users
```
Comment (josegonzalez) [inline Vagrantfile]: Note: If anything this should either:

- be reversed
- just autodetect if you are on windows and don't sync the directory in that case.

File: bootstrap.sh (hunk 1 above)
Comment (vijayviji) [inline bootstrap.sh]: 'apt-get install dokku' was prompting a question. So, I changed it. This is not relevant to the issue. But I faced with this while debugging.

File: Vagrantfile (hunk 2 above)
Comment (vijayviji) [inline Vagrantfile]: Sure, I can change it.

File: bootstrap.sh (hunk 1 above)
Comment (josegonzalez) [inline bootstrap.sh]: Gotcha. I think this was fixed already in a separate PR, so if you can rebase onto master, that would be :+1:

**Comment by vijayviji** [comment]
Hi Jose. I rebased the master.

**Comment by josegonzalez** [comment]
Can you squash your commits into one?

**Comment by vijayviji** [comment]
I rebased and squashed.

**Comment by josegonzalez** [comment]
Can you update the virtualbox documentation with a note at the top stating how it works on windows?
